libafl/src/observers/map/const_map.rs

```diff
@@ -150,37 +148,30 @@ where
     }
 }
 
-impl<T, const N: usize> VariableLengthMapObserver for ConstMapObserver<'_, T, N>
+impl<T, const N: usize> ConstantLengthMapObserver<N> for ConstMapObserver<'_, T, N>
```

Comment:
How about renaming them to `ConstLenMapObserver` and `VarLenMapObserver` btw? Seems shorter and just as to clear.
Random idea of course, feel free to ignore.